docs/LLS_Metallicities-Wotta/mcmc_met.py

```diff
@@ -0,0 +1,258 @@
+"""
+Module for analyzing metallicity with MCMC
+
+"""
+
+
+import numpy as np
+import sys
+import os
+
+from astropy.table import Table
+from pyigm.metallicity.mcmc import mcmc_ions
+
+
+################################################
+################################################
+
+
+def run_full_mcmc(infile='MgIIsurvey_MTL.ascii', nthread=4, met_guess='-1.0', dens_guess='-2.5', carbalpha_guess='0.0', carbalpha_use=True, logUconstraint=False,UVB='HM05'):
+    
+    """ Run MCMC
+
+    Parameters
+    ---------
+    nthread : int, optional
+      Number of threads per system
+
+    """
+    
+    ########################    
+    ##CBW adjust parameters
+    ########################
+    
+    ##NOTE: In mcmc.py, we have the following lines:
+    ##  > if not testing:
+    ##  >     nwalkers=np.max([nwalkers,400])
+    ##  >     nsamp=np.max([nsamp,100])
+    ##which means the options here may be overridden later
+    ##(depending on your choices here).
+    
+    nwalkers = int(400) ##make sure it's an integer
+    nsteps = int(400) ##make sure it's an integer
+    testing = False
+    optim = 'guess' ##or False
+    outsave = 'MCMC_FULL'
+    
+    ##Grid file
+    ##Test whether to include the carbalpha parameter
+    if str(carbalpha_use).lower() == 'false':

```

Comment:
We should think of a nicer way of sharing the Cloudy grids (they are too big for pyigm).  Could stage on my machine in UCSC (next to igmspec).

Thoughts @mifumagalli ??